cico_run_tests.sh

```diff
@@ -32,16 +32,20 @@ make docker-start
 make docker-deps
 make docker-generate
 make docker-build
-make docker-test-unit
 
+# Run the test without generating coverage information (for faster feedback
+# cycles).
+make docker-test-unit-no-coverage
 make integration-test-env-prepare
-
 function cleanup {
   make integration-test-env-tear-down
 }
 trap cleanup EXIT
-
 make docker-test-migration
+make docker-test-integration-no-coverage
+
+# Run the tests that generate coverage information
+make docker-test-unit
```

Comment:
We should probably split this out in it's own file. We get the quick feedback now, but still require the whole job to finish before being allowed to merge the pr.